.github/workflows/create-release.yml

```diff
@@ -12,32 +12,19 @@ jobs:
     outputs:
       release_notes: ${{ steps.notes.outputs.body }}
     steps:
+    - name: Setup Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15
 
-    - name: Install Pack
-      run: |
-        id=$(docker create buildpacksio/pack:0.14.2)
-        sudo docker cp $id:/layers/paketo-buildpacks_go-build/targets/bin/pack /usr/local/bin/
-        docker rm -v $id
+    - name: Install Global Pack
+      uses: buildpacks/github-actions/setup-pack@main
```

Comment:
To ensure that the same version of pack is used to test the builder and create the shipped artifact, please see this PR on base builder: https://github.com/paketo-buildpacks/base-builder/pull/55 which uses a file`.github/pack-version` to keep track of the version and use it in testing and workflows.